.nvmrc

```diff
@@ -0,0 +1 @@
+18.17.1
```

Comment:
Maybe just `18`?

Reply:
There are a few libraries that are pinned lower than they could be because they require a higher minor verion of node. So the specific pin to `18.17.1` is necessary here.